.changeset/nasty-lemons-bow.md

```diff
@@ -0,0 +1,7 @@
+---
+"@firebase/auth-compat": patch
```

Comment:
No changeset needed for exp packages, this file can be removed.

Reply:
```suggestion
```